assets/sass/v2/_device-challenge.scss

```diff
@@ -49,6 +49,13 @@
     }
     .app-link-content {
       margin-bottom: 15px;
+
+      .spinner {
+        margin-bottom: 8px;
```

Comment:
does the look okay on smaller screens? wonder how to decide `8px`

Reply:
It's to make the top and bottom margin equivalent.